src/_common/game/screenshot/screenshot.model.ts

```diff
@@ -11,9 +11,15 @@ export class GameScreenshot extends Model {
 	posted_on!: number;
 	status!: number;
 	media_item!: MediaItem;
-	img_thumbnail!: string;
-	img_thumbnail_med!: string;
-	img_thumbnail_large!: string;
+
+	// Needed for lightbox and other components that are used together with GameVideo.
+	get img_thumbnail() {
+		return this.media_item.mediaserver_url;
+	}
+
+	set img_thumbnail(img: string) {
```

Comment:
You don't need the setters for these, right?

Reply:
We do, through Object.assign in forms we assign this back onto the model.